docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -0,0 +1,172 @@
+[id="migrating-the-rgw-backends_{context}"]
+
+= Migrating the {Ceph} RGW backends
+
+To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_{context}[{Ceph} daemon cardinality].
+
+//kgilliga: Should each instance of "CephStorage" node be changed to "Red Hat Ceph Storage node" (downstream) and "Ceph Storage node" (upstream)?
+
+.Procedure
+
+. Add the RGW label to the {Ceph} nodes:
++
+----
+for i in 0 1 2; {
+    ceph orch host label add cephstorage-$i rgw;
+}
+----
++
+----
+[ceph: root@controller-0 /]#
+
+for i in 0 1 2; {
+    ceph orch host label add cephstorage-$i rgw;
+}
+
+Added label rgw to host cephstorage-0
+Added label rgw to host cephstorage-1
+Added label rgw to host cephstorage-2
+
+[ceph: root@controller-0 /]# ceph orch host ls
+
+HOST       	ADDR       	LABELS      	STATUS
+cephstorage-0  192.168.24.54  osd rgw
+cephstorage-1  192.168.24.44  osd rgw
+cephstorage-2  192.168.24.30  osd rgw
+controller-0   192.168.24.45  _admin mon mgr
+controller-1   192.168.24.11  _admin mon mgr
+controller-2   192.168.24.38  _admin mon mgr
+
+6 hosts in cluster
+----
+
+. During the overcloud deployment, RGW is applied at step2
+(external_deployment_steps), and a cephadm compatible spec is generated in
+`/home/ceph-admin/specs/rgw` from the https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/ansible_plugins/modules/ceph_mkspec.py[ceph_mkspec] ansible module.
```

Comment:
remove the github link downstream and just state that a spec is generated in `/home/ceph-admin/specs/rgw` from director